drivers/pure/flasharray/volumes.go

```diff
@@ -4,28 +4,13 @@ type VolumeServices struct {
 	client *Client
 }
 
-func (fs *VolumeServices) ListAllAvailableVolumes(params map[string]string, data interface{}) ([]Volumes, error) {
-	params["destroyed"] = "false"
-	req, err := fs.client.NewRequest("GET", "volumes", params, data)
+func (vols *VolumeServices) ListAllAvailableVolumes(params map[string]string, data interface{}) ([]VolResponse, error) {
+	req, err := vols.client.NewRequest("GET", "volumes", params, data)
```

Comment:
I see in function ListAllDestroyedVolumesFromFA you are passing params as nil. is that fine here?